<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>time_setting</class>
 <widget class="QWidget" name="time_setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>800</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_backdrop">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>720</width>
     <height>360</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border-image: url(:/tu/ALL-IN-ONE ICON/BKG_long.png);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Label_return_time" name="label_return">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>500</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Label_save_time" name="label_save">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>500</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_num">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>720</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>时间设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Label_year1_time" name="label_year1">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>360</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);
border-image: url(:/tu/ALL-IN-ONE ICON/BKG_long_small.png);</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="Label_year2_time" name="label_year2">
   <property name="geometry">
    <rect>
     <x>417</x>
     <y>360</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);
border-image: url(:/tu/ALL-IN-ONE ICON/BKG_long_small.png);</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="Label_year3_time" name="label_year3">
   <property name="geometry">
    <rect>
     <x>479</x>
     <y>360</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);
border-image: url(:/tu/ALL-IN-ONE ICON/BKG_long_small.png);</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="Label_year4_time" name="label_year4">
   <property name="geometry">
    <rect>
     <x>541</x>
     <y>360</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);
border-image: url(:/tu/ALL-IN-ONE ICON/BKG_long_small.png);</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="Label_month_time" name="label_month">
   <property name="geometry">
    <rect>
     <x>641</x>
     <y>360</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);
border-image: url(:/tu/ALL-IN-ONE ICON/BKG_long_small.png);</string>
   </property>
   <property name="text">
    <string>03</string>
   </property>
  </widget>
  <widget class="Label_day_time" name="label_day">
   <property name="geometry">
    <rect>
     <x>703</x>
     <y>360</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);
border-image: url(:/tu/ALL-IN-ONE ICON/BKG_long_small.png);</string>
   </property>
   <property name="text">
    <string>21</string>
   </property>
  </widget>
  <widget class="Label_hour_time" name="label_hour">
   <property name="geometry">
    <rect>
     <x>803</x>
     <y>360</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);
border-image: url(:/tu/ALL-IN-ONE ICON/BKG_long_small.png);</string>
   </property>
   <property name="text">
    <string>11</string>
   </property>
  </widget>
  <widget class="Label_minute_time" name="label_minute">
   <property name="geometry">
    <rect>
     <x>865</x>
     <y>360</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);
border-image: url(:/tu/ALL-IN-ONE ICON/BKG_long_small.png);</string>
   </property>
   <property name="text">
    <string>00</string>
   </property>
  </widget>
  <widget class="Label_up1_time" name="label_up1">
   <property name="geometry">
    <rect>
     <x>448</x>
     <y>295</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);
border-image: url(:/tu/ALL-IN-ONE ICON/BKG_long_small.png);</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="Label_up2_time" name="label_up2">
   <property name="geometry">
    <rect>
     <x>672</x>
     <y>295</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);
border-image: url(:/tu/ALL-IN-ONE ICON/BKG_long_small.png);</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="Label_up3_time" name="label_up3">
   <property name="geometry">
    <rect>
     <x>832</x>
     <y>295</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);
border-image: url(:/tu/ALL-IN-ONE ICON/BKG_long_small.png);</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="Label_down1_time" name="label_down1">
   <property name="geometry">
    <rect>
     <x>448</x>
     <y>425</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);
border-image: url(:/tu/ALL-IN-ONE ICON/BKG_long_small.png);</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="Label_down2_time" name="label_down2">
   <property name="geometry">
    <rect>
     <x>672</x>
     <y>425</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);
border-image: url(:/tu/ALL-IN-ONE ICON/BKG_long_small.png);</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="Label_down3_time" name="label_down3">
   <property name="geometry">
    <rect>
     <x>832</x>
     <y>425</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);
border-image: url(:/tu/ALL-IN-ONE ICON/BKG_long_small.png);</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Label_return_time</class>
   <extends>QLabel</extends>
   <header>time_setting.h</header>
  </customwidget>
  <customwidget>
   <class>Label_save_time</class>
   <extends>QLabel</extends>
   <header>time_setting.h</header>
  </customwidget>
  <customwidget>
   <class>Label_year1_time</class>
   <extends>QLabel</extends>
   <header>time_setting.h</header>
  </customwidget>
  <customwidget>
   <class>Label_year2_time</class>
   <extends>QLabel</extends>
   <header>time_setting.h</header>
  </customwidget>
  <customwidget>
   <class>Label_year3_time</class>
   <extends>QLabel</extends>
   <header>time_setting.h</header>
  </customwidget>
  <customwidget>
   <class>Label_year4_time</class>
   <extends>QLabel</extends>
   <header>time_setting.h</header>
  </customwidget>
  <customwidget>
   <class>Label_month_time</class>
   <extends>QLabel</extends>
   <header>time_setting.h</header>
  </customwidget>
  <customwidget>
   <class>Label_day_time</class>
   <extends>QLabel</extends>
   <header>time_setting.h</header>
  </customwidget>
  <customwidget>
   <class>Label_hour_time</class>
   <extends>QLabel</extends>
   <header>time_setting.h</header>
  </customwidget>
  <customwidget>
   <class>Label_minute_time</class>
   <extends>QLabel</extends>
   <header>time_setting.h</header>
  </customwidget>
  <customwidget>
   <class>Label_up1_time</class>
   <extends>QLabel</extends>
   <header>time_setting.h</header>
  </customwidget>
  <customwidget>
   <class>Label_up2_time</class>
   <extends>QLabel</extends>
   <header>time_setting.h</header>
  </customwidget>
  <customwidget>
   <class>Label_up3_time</class>
   <extends>QLabel</extends>
   <header>time_setting.h</header>
  </customwidget>
  <customwidget>
   <class>Label_down1_time</class>
   <extends>QLabel</extends>
   <header>time_setting.h</header>
  </customwidget>
  <customwidget>
   <class>Label_down2_time</class>
   <extends>QLabel</extends>
   <header>time_setting.h</header>
  </customwidget>
  <customwidget>
   <class>Label_down3_time</class>
   <extends>QLabel</extends>
   <header>time_setting.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
